### ISSUE TITLE
 TFException: NodeDef mentions attr 'output_dtype' not in Op<name=Multinomial; signature=logits:T, num_samples:int32 -> output:int64; attr=seed:int,default=0; attr=seed2:int,default=0; attr=T:type,allowed=[DT_FLOAT, DT_DOUBLE, DT_INT32, DT_INT64, DT_UINT8, DT_INT16, DT_INT8, DT_UINT16, DT_HALF]; is_stateful=true>; NodeDef: multinomial_3/Multinomial = Multinomial[T=DT_FLOAT, output_dtype=DT_INT64, seed=6006, seed2=233](Log_3, multinomial_3/Multinomial/num_samples).

### ISSUE THREAD

**Issue by husnainyousaf88** [Unity-Technologies/ml-agents#1551]
using unity 2018.2.19
tensorflow 1.7.1
ml-agents v0.5
this error come only on running pretrained model.
tried to upgrade or degrade versions of tfsharp but that results crashing unity on pressing playing button.

TFException: NodeDef mentions attr 'output_dtype' not in Op<name=Multinomial; signature=logits:T, 
num_samples:int32 -> output:int64; attr=seed:int,default=0; attr=seed2:int,default=0; attr=T:type,allowed=[DT_FLOAT, DT_DOUBLE, DT_INT32, DT_INT64, DT_UINT8, DT_INT16, DT_INT8, DT_UINT16, DT_HALF]; is_stateful=true>; NodeDef: multinomial_3/Multinomial = Multinomial[T=DT_FLOAT, output_dtype=DT_INT64, seed=6006, seed2=233](Log_3, multinomial_3/Multinomial/num_samples). (Check whether your GraphDef-interpreting binary is up to date with your GraphDef-generating binary.).
TensorFlow.TFStatus.CheckMaybeRaise (TensorFlow.TFStatus incomingStatus, System.Boolean last) (at <1fe2de69842a4a4ba15256b83cca05f3>:0)
TensorFlow.TFGraph.Import (TensorFlow.TFBuffer graphDef, TensorFlow.TFImportGraphDefOptions options, TensorFlow.TFStatus status) (at <1fe2de69842a4a4ba15256b83cca05f3>:0)
TensorFlow.TFGraph.Import (System.Byte[] buffer, TensorFlow.TFImportGraphDefOptions options, TensorFlow.TFStatus status) (at <1fe2de69842a4a4ba15256b83cca05f3>:0)
TensorFlow.TFGraph.Import (System.Byte[] buffer, System.String prefix, TensorFlow.TFStatus status) (at <1fe2de69842a4a4ba15256b83cca05f3>:0)
MLAgents.CoreBrainInternal.InitializeCoreBrain (MLAgents.Batcher brainBatcher) (at Assets/ML-Agents/Scripts/CoreBrainInternal.cs:133)
MLAgents.Brain.InitializeBrain (MLAgents.Academy aca, MLAgents.Batcher brainBatcher) (at Assets/ML-Agents/Scripts/Brain.cs:207)
MLAgents.Academy.InitializeEnvironment () (at Assets/ML-Agents/Scripts/Academy.cs:288)

**Comment by mmattar** [comment]
@husnainyousaf88 - which scene are you trying to run?

**Comment by mmattar** [comment]
Hi @husnainyousaf88 - I'm closing this issue for now, but if you're able to provide more context please submit a new Issue. Thanks.
